assembly/common/src/main/resources/bin/sqlshell.sh

```diff
@@ -149,22 +149,19 @@ if [[ "$jversion" < "1.8" ]]; then
    exit 1
 fi
 
-# set splice lib dir
-SPLICE_LIB_DIR="##SPLICELIBDIR##"
-
-if [[ "$SPLICE_LIB_DIR" == *"##"* ]]; then
-   CURDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-
-   # ends in bin, go up and over to lib
-   if [[ "$CURDIR" = *"bin" ]]; then
-      SPLICE_LIB_DIR="${CURDIR}/../lib"
-   else #subdir
-      SPLICE_LIB_DIR="${CURDIR}/lib"
-   fi
-fi
-
 # set up classpath to point to splice jars
-export CLASSPATH="${SPLICE_LIB_DIR}/*"
+IJ_SYS_ARGS="-Djdbc.drivers=com.splicemachine.db.jdbc.ClientDriver"
+CURDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+DBCLIENT=$(find ${CURDIR}/lib/ -type f -name 'db-client*-SNAPSHOT.jar')
+DBTOOLS=$(find ${CURDIR}/lib/ -type f -name 'db-tools*-SNAPSHOT.jar')
+DBENGINE=$(find ${CURDIR}/lib/ -type f -name 'db-engine*-SNAPSHOT.jar')
+
+#check if all the tools needed to connect are found
+if [ -z "$DBCLIENT"] || [ -z "$DBTOOLS"] || [ -z "$DBENGINE"]; then
```

Comment:
i like this added check! perhaps we use the old 'SPLICE_LIB_DIR' to search for these files?